comms/AttributeImpl.h

```diff
@@ -19,7 +19,7 @@ class CAttributeBase : public clib::CLockableUnknown
     virtual const TCHAR *GetText(bool refresh, bool noBroadcast = false) const = 0;
     virtual const TCHAR *GetLabel() const = 0;
 
-    bool pluginFolderExists(const std::string & batchFile, boost::filesystem::path & foundFolder, int level, bool pluginFolder) const
+    bool pluginFolderExists(const std::string & batchFile, boost::filesystem::path & foundFolder, int level, bool typeFolder, const std::string & subFolder, const std::string & subSubFolder) const
```

Comment:
@rpastrana I did it this way so that it is not dependent on the local file structure. I thought this less convoluted than constructing the path using path variables before calling it. I can change that if you think it better. All of this will most likely change given we need to do things more "generic" in version 7.